command/secrets_enable.go

```diff
@@ -229,6 +230,14 @@ func (c *SecretsEnableCommand) Flags() *FlagSets {
 			"each time with 1 key.",
 	})
 
+	f.StringSliceVar(&StringSliceVar{
+		Name:   flagNameDelegatedAuthAccessors,
+		Target: &c.flagDelegatedAuthAccessors,
+		Usage: "A list of permitted authentication accessors this backend can delegate authentication to. " +
```

Comment:
Does it matter that the usage here does not exactly match the documentation?

Reply:
Oops yup the docs are wrong, it only works by passing the argument multiple times. Thanks for catching that!